scripts/abby_excel.py

```
@@ -0,0 +1,183 @@
+assets_dir = current_dir / "assets"
+outputs = [p for p in assets_dir.iterdir() if p.is_dir() if os.path.basename(str(p)) != "__pycache__"]
+
+outsheet = JsonItem([])
+outsheet[2][0] = 'target'
+outsheet[2][1] = 'target-display-name'
+outsheet[2][2] = 'type'
+outsheet[2][3] = 'color'
+
+dataset_y = {}
+y = 4
+for dataset in dataset_sizes.keys():
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)